experimental/devices/mfrc522/rf522.go

```diff
@@ -0,0 +1,764 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package mfrc522 controls a Mifare RFID card reader.
+// The code is largely ported from Python library : https://github.com/mxgxw/MFRC522-python
```

Comment:
Keep an empty line after the first line.
I'd remove the python reference.
Use the title formatting. For example, https://github.com/google/periph/blob/master/devices/bmxx80/bmxx80.go#L5 becomes https://periph.io/x/periph/devices/bmxx80, which is nice.